spec.bs

```diff
@@ -2311,10 +2422,10 @@ Security considerations {#security-considerations}
 Same-origin policy {#same-origin-policy}
 ----------------------------------------
 
-Writes to the [=aggregatable report cache=], [=context ID map=], [=contribution
-cache=] and [=debug scope map=] are separated by the reporting [=origin=] and
-the data included in any report with a given reporting [=origin=] are generated
-with only data from that [=origin=].
+Writes to the [=aggregatable report cache=], [=contribution cache=], [=debug
```

Comment:
I'm not sure how much I like this statement, it's true every report has an origin, each cache entry has a debug and batching scope, and the other two are mapped to unique scopes.

When I dig into the details it does seem that the underlying scopes or operations are attributed to a given origin, but I'm not sure they are separated by origin exactly. As the current storage model stands, one has the information needed to separate them by origin but the lists at least are not really separated at all. 

Anyway, this is a long way to say I think 'attributed to' is a bit more correct than 'separated by'.